src/App.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import './App.scss';
+
+class App extends React.Component {
+  state = {
+    count: 0,
+  }
+
+  addOne = () => {
+    this.setState(state => ({
+      count: state.count + 1,
+    }));
+  };
+
+  add100 = () => {
+    this.setState(state => ({
+      count: state.count + 100,
+    }));
+  };
+
+  increase = () => {
+    if (this.state.count % 5 === 0) {
+      this.add100();
+      this.addOne();
+    } else {
+      this.addOne();
```

Comment:
Ти можеш винести **this.addOne** з іфу, так як він в тебе в будь-якому випадку виконується

```suggestion
    if (this.state.count % 5 === 0) {
      this.add100();
    }
      this.addOne();
```

Reply:
согласна. addOne можно вызывать вне ифа и убрать else